editors/liveview/assets/css/app.css

```
@@ -3,3 +3,551 @@
+.field {
+  border: 1px solid var(--line); background: var(--code-bg); border-radius: 6px;
+  padding: 6px 9px; font-family: var(--code-font, ui-monospace, monospace); font-size: 12.5px;
+  color: var(--ink); outline: none;
+}
+.field:focus { border-color: var(--accent); }
+
+/* result / output / error blocks */
+.io-block {
+  margin: 0 0 8px 0; padding: 9px 12px; border: 1px solid var(--line);
+  border-radius: 7px; background: var(--code-bg); font-family: var(--code-font, ui-monospace, monospace);
+  font-size: 12.5px; white-space: pre-wrap; word-break: break-word; color: var(--ink);
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)